packages/helper-functions/src/lib/readLocalFile.ts

```diff
@@ -0,0 +1,19 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export const readLocalFile = (
```

Comment:
This function doesn't look good - hardcoded values are used. We need to find another way to read files.